docs/content/documentation/getting-started/directory-structure.md

```diff
@@ -49,3 +49,6 @@ and available variables.
 ## `themes`
 Contains themes that can be used for your site. If you are not planning to use themes, leave this directory empty.
 If you want to learn about themes, see the [themes documentation](@/documentation/themes/_index.md).
+
+## `public`
+Contains the built site. Might appear from [`zola build`](@/documentation/getting-started/cli-usage.md#build) and [`zola serve`](@/documentation/getting-started/cli-usage.md#serve) depending on your configuration.
```

Comment:
Can you move that text around L20 and remove the header? Also

> Contains the built site

is technically incorrect for `zola serve` so I would remove it.

Something like:

You might also see a `public` directory if you are running the default `zola build/serve` commands which contains some output for the site: the full site for `zola build` and only the static assets for `zola serve`. This folder will be deleted/created automatically by `zola serve`.

Reply:
Fixed!